src/contconic.jl

```diff
@@ -1,6 +1,16 @@
 using MathOptInterfaceUtilities # Defines getindex for VectorAffineFunction
 const MOIU = MathOptInterfaceUtilities
 
+_vovf(vecofvars) = vecofvars ? MOI.VectorOfVariables : MOI.VectorAffineFunction{Float64}
+function _addvovconstraint!(instance, v::Vector{MOI.VariableIndex}, set::MOI.AbstractVectorSet, vecofvars::Bool)
```

Comment:
Please document all helper functions. The code here needs to be easy for solver authors to read.